scripts/templates/ComponentName.doc.js

```diff
@@ -1,35 +1,167 @@
 // @flow strict
 import React, { type Node } from 'react';
+import { Box } from 'gestalt';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
+import CombinationNew from './components/CombinationNew.js';
 import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
-card(<PageHeader name="ComponentName" description="ComponentName allows users to ..." />);
+card(<PageHeader name="Component Name" description="Brief description of this cmp" />);
 
 card(
-  <Example
-    name="Basic Example"
-    defaultCode={`
-<ComponentName>
-
-</ComponentName>
+  <MainSection name="Examples" showHeading={false}>
+    <MainSection.Subsection>
+      <MainSection.Card
+        title="Optional Card Title"
+        description={`
+           Optional description about this \`Example\`
+        `}
+        cardSize="lg"
+        defaultCode={`
+Code for this example goes here
 `}
-  />,
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
+// Specifying Component gives auto warnings about missing props
 card(
   <PropTable
+    Component={Box}
     props={[
       {
-        name: 'accessibilityLabel',
-        type: 'string',
-        description: 'String that clients such as VoiceOver will read to describe the element.',
+        name: 'dummyProp',
+        type: 'dummyType',
+        defaultValue: 'dummyDefault',
+        description: '[Briefly describe the goal of this prop]',
+        href: '[Name of the section demonstrating this prop]',
       },
     ]}
   />,
 );
 
+card(
+  <MainSection name="Best Practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Description about what you should Do"
+        defaultCode={`
+Code for this example goes here
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description={`
+          What not to do goes \`here\`. Using backticks instead of quotes allows you to use [Markdown]("https://www.markdownguide.org/")
```

Comment:
Can you add this note about backticks to all fields where it's relevant? Or we could add a note at the top of the file if it's going to be too repetitive.

EDIT: oh, I think you did already. Does it also need to be added to the "do" description above?

Reply:
Yes, good catch!